src/main/core.js

```diff
@@ -59,6 +51,7 @@ function attachComments(text, ast, opts) {
     comments.attach(astComments, ast, text, opts);
   }
   ast.tokens = [];
+  opts.comments = astComments || [];
```

Comment:
Any better idea to access all comments?

Reply:
I think it's fine to keep it in `options`, but let's use a prefixed key name (`__comments`) or a symbol. Bad things might happen if a plugin has an option named `comments`.